_install-and-configure/upgrade-opensearch/appendix/rolling-upgrade-lab.md

```diff
@@ -0,0 +1,887 @@
+      -key /usr/share/opensearch/config/admin-key.pem
+   ```
+   {% include copy.html %}
+   <p class="codeblock-label">Example response</p>
+   ```bash
+   Security Admin v7
+   Will connect to localhost:9300 ... done
+   Connected as CN=A,OU=DOCS,O=OPENSEARCH,L=PORTLAND,ST=OREGON,C=US
+   OpenSearch Version: 1.3.7
+   OpenSearch Security Version: 1.3.7.0
+   Contacting opensearch cluster 'opensearch' and wait for YELLOW clusterstate ...
+   Clustername: opensearch-dev-cluster
+   Clusterstate: GREEN
+   Number of nodes: 4
+   Number of data nodes: 4
+   .opendistro_security index already exists, so we do not need to create one.
+   Will retrieve '/config' into /usr/share/opensearch/backups/config.yml 
+      SUCC: Configuration for 'config' stored in /usr/share/opensearch/backups/config.yml
+   Will retrieve '/roles' into /usr/share/opensearch/backups/roles.yml 
+      SUCC: Configuration for 'roles' stored in /usr/share/opensearch/backups/roles.yml
+   Will retrieve '/rolesmapping' into /usr/share/opensearch/backups/roles_mapping.yml 
+      SUCC: Configuration for 'rolesmapping' stored in /usr/share/opensearch/backups/roles_mapping.yml
+   Will retrieve '/internalusers' into /usr/share/opensearch/backups/internal_users.yml 
+      SUCC: Configuration for 'internalusers' stored in /usr/share/opensearch/backups/internal_users.yml
+   Will retrieve '/actiongroups' into /usr/share/opensearch/backups/action_groups.yml 
+      SUCC: Configuration for 'actiongroups' stored in /usr/share/opensearch/backups/action_groups.yml
+   Will retrieve '/tenants' into /usr/share/opensearch/backups/tenants.yml 
+      SUCC: Configuration for 'tenants' stored in /usr/share/opensearch/backups/tenants.yml
+   Will retrieve '/nodesdn' into /usr/share/opensearch/backups/nodes_dn.yml 
+      SUCC: Configuration for 'nodesdn' stored in /usr/share/opensearch/backups/nodes_dn.yml
+   Will retrieve '/whitelist' into /usr/share/opensearch/backups/whitelist.yml 
+      SUCC: Configuration for 'whitelist' stored in /usr/share/opensearch/backups/whitelist.yml
+   Will retrieve '/audit' into /usr/share/opensearch/backups/audit.yml 
+      SUCC: Configuration for 'audit' stored in /usr/share/opensearch/backups/audit.yml
+   ```
+1. **Optional**: Create a backup directory for TLS certificates and store copies of the certificates. Repeat this for each node if you use unique TLS certificates:
+   ```bash
+   mkdir /usr/share/opensearch/backups/certs && cp /usr/share/opensearch/config/*pem /usr/share/opensearch/backups/certs/
+   ```
+   {% include copy.html %}
+1. Terminate the pseudo-TTY session:
+   ```bash
+   exit
+   ```
+   {% include copy.html %}
+1. Copy the files to your host:
+   ```bash
+   docker cp os-node-01:/usr/share/opensearch/backups ~/deploy/
+   ```
+   {% include copy.html %}
+
+## Performing the upgrade
+
+Now that the cluster is configured, and you made backups of important files and settings, it's time to begin the version upgrade.
```

Comment:
This might be in a different section of the upgrade docs, but are there any restrictions when it comes to which version I can upgrade to? For example, can I use these steps to upgrade from 1.3 to 2.6?

With that question in mind, let's call out the versions we're using in this lab.

Reply:
```suggestion
Now that the cluster is configured, and you made backups of important files and settings, it's time to begin the version upgrade from 1.3.7 to 2.5.0."
```

Reply:
Yes it is included `_install-and-configure/upgrade-opensearch/rolling-upgrade.md`